pkgs/development/python-modules/pytrends/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, requests
+, lxml
+, pandas
+}:
+
+buildPythonPackage rec {
+  pname = "pytrends";
+  version = "4.6.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "03gnn2mgjvpc7pbcijy7xilrhgjg7x2pp6ci96pdyqnhkqv02d3n";
+  };
+
+  # Disabling integration tests
```

Comment:
This is just a nit, but I feel like comments like this add nothing of value. It says the same thing as `doCheck = false;` :smile: 
I'd either leave it out or explain briefly the failure that makes it necessary.

Reply:
I agree with you, but someone has asked me before to always comment why I disable tests :) but in this case it's probably obvious.